cypress/e2e/dashboard/chapters/[chapterId]/edit.cy.ts

```diff
@@ -73,4 +73,28 @@ describe('chapter edit dashboard', () => {
       cy.contains(chapterData.name);
     });
   });
+
+  it('deny admins to delete a chapter', () => {
+    const chapterId = 1;
+    cy.login(Cypress.env('JWT_ADMIN_USER'));
+
+    cy.updateChapter(chapterId, chapterData).then((response) => {
+      expectToBeRejected(response);
+
+      cy.visit(`/dashboard/chapters/${chapterId}`);
+      cy.findByRole('button', { name: 'Delete Chapter' })
+        .click()
+        .findByRole('button', { name: 'Are you sure?' })
+        .click()
+        .findByRole('button', { name: 'Delete' })
+        .click();
+    });
+
+    cy.updateChapter(chapterId, chapterData).then((response) => {
+      expect(response.status).to.eq(200);
+
+      cy.visit(`/dashboard/chapters/${chapterId}`);
+      cy.contains(chapterData.name);
+    });
```

Comment:
```suggestion
    cy.login();
    cy.deleteChapter(chapterId).then((response) => {
      expect(response.status).to.eq(200);
      expect(response.body.errors).not.to.exist;
    });
```
We're only testing deletion... and I needed somewhere to put the owner request!